tests/test_data/test_config/test_elasticsearch.yaml

```diff
@@ -0,0 +1,8 @@
+core_lib:
+  elasticsearch:
+    _target_: core_lib.connection.elasticsearch_connection_registry.ElasticSearchConnectionRegistry
+    config:
+      url:
+        protocol: http
+        host: ${oc.env:ELASTIC_HOST}

```

Comment:
let's add a test string over here and remove the environment variables for the test config